app/src/main/java/com/calendar/activity/CreateEventActivity.java

```diff
@@ -70,23 +72,36 @@ public void showAdvancedOptions(View view) {
             annualCheckBox.setVisibility(View.VISIBLE);
             reminderCheckBox.setVisibility(View.VISIBLE);
             freeCheckBox.setVisibility(View.VISIBLE);
+            showReminderTimeCheck();
+            showReminderClockCheck();
         } else {
             annualCheckBox.setVisibility(View.INVISIBLE);
             reminderCheckBox.setVisibility(View.INVISIBLE);
             freeCheckBox.setVisibility(View.INVISIBLE);
+            reminderTimeCheckBox.setVisibility(View.INVISIBLE);
+            timePicker.setVisibility(View.INVISIBLE);
         }
     }
 
     public void showReminderTimeCheckBox(View view) {
```

Comment:
Can be private?

Reply:
No it can, becouse it is connected to button